client/src/components/Tea/TeaDetails/TeaDetailsContainer.jsx

```diff
@@ -11,30 +11,41 @@ class TeaDetailsContainer extends Component {
     this.props.updateFlash(status);
   };
 
+  componentDidMount() {
+    this.props.getTeaList(this.props.userID);
+  }
+
   render() {
-    return (
+    return !this.props.tea ? null : (
```

Comment:
This method will never return `null`, as `this.props.tea` is guaranteed to either be a value within `allTeas`, or `emptyObj`. It's probably a better idea not to pass in an empty object. You should consider not short-circuiting with `emptyObj` when `state.teas.allTeas[ownProps.match.params.id]` is absent.

Reply:
makes sense. this change makes `emptyObj` obsolete both here and on TeaEditorContainer.js